meta/engineering_principles.md

```diff
@@ -0,0 +1,137 @@
+# Engineering Principles
+
+We have [company values](what_is_artsy.md), and are working on our shared values with design and product. However,
+when we make decisions on what and how to do engineering we need guiding principals. Things that make Artsy's
+development culture unique and drive us to do great work.
+
+### Open Source by Default
+
+We consider the data and relationships that make Artsy difficult to copy, code only represents the past and
+close-to present. A code project should start as open, then move to closed if there are valid business needs that
+need to be hidden.
```

Comment:
I'm not sure this is a viable business practice. Launching projects as open-source means associating a license with the content and potentially accepting external contributions. Making projects private later is not as simple as checking a box and removing the license file; it's unclear whether we even _could_ make outside contributions private without explicitly re-licensing that content. Plus, any external forks of the previously-public project become orphaned or worse--there becomes a new public ["parent" repo](https://help.github.com/articles/what-happens-to-forks-when-a-repository-is-deleted-or-changes-visibility/#changing-a-public-repository-to-a-private-repository) that we don't control.

Reply:
It actually is as simple as ticking a box, because MIT licensed code may be used in closed source projects. The only thing that needs to happen is to not remove the license _or_ annotate the code from external contributors, which could easily be automated as that data exists in SCM anyways.

As for there being a new public parent repo, while that could happen it’s extremely unlikely to ever happen to a repo of ours and additionally isn’t really any different from people having local clones. It’s not something I would really consider in this debate.